packages/jest-config/src/constants.ts

```diff
@@ -8,7 +8,7 @@
 import * as path from 'path';
 
 export const NODE_MODULES = `${path.sep}node_modules${path.sep}`;
-export const DEFAULT_JS_PATTERN = '\\.[jt]sx?$';
+export const DEFAULT_JS_PATTERN = '\\.m?[jt]sx?$';
```

Comment:
this should not be included

Reply:
Fixed. I looked from backend development perspective, forgetting about JSX.